documentation/user_docs/docs/images/Head_output_table.csv

```diff
@@ -0,0 +1,23 @@
+Scientific view:  ``head(file);`` ,Programmer view:  ``struc = head(file)``
```

Comment:
Having looked at the file preview, it is very confusing as it stands. I keep thinking that there is a correlation between the items in each row, but there isn't. Could you break this into 2 tables, one with just the first column, one with just the second. Thanks

Reply:
There is some correlation between rows but you are probably right about confusion